locales/fr.js

```diff
@@ -0,0 +1,5 @@
+{
+    "Welcome":"Bienvenue à Barefoot Nomad",
+    "NotFound":"Pas trouvé",
+    "AppDesc":"Rendre les voyages et l'hébergement en entreprise faciles et pratiques"
+}
```

Comment:
A new line is needed at the of the file

Reply:
Thank you @AbdoulNuru . Allow me to fix those ESlint issues.